connectors/sync_job_runner.py

```diff
@@ -374,11 +376,15 @@ async def _sync_done(self, sync_status, sync_error=None):
 
         self.sync_job.log_info(
             f"Sync ended with status {sync_status.value} -- "
-            f"created: {result.get('doc_created', 0)} | "
-            f"updated: {result.get('doc_updated', 0)} | "
-            f"deleted: {result.get('doc_deleted', 0)} "
+            f"created: {ingestion_stats.get('doc_created', 0)} | "
+            f"updated: {ingestion_stats.get('doc_updated', 0)} | "
+            f"deleted: {ingestion_stats.get('doc_deleted', 0)} "
             f"(took {int(time.time() - self._start_time)} seconds)"  # pyright: ignore
         )
+        self.sync_job.log_info("--- Counters ---")
```

Comment:
I think it's worth moving the logging logic ([L384-L387](https://github.com/elastic/connectors/pull/2323/files#diff-f4def1564167603a89d66578a9ac61c61f5d0c63298e1ceab10b22f3d949ff8eR384-R387)) out of the _sync_done method just to keep it simple.

Reply:
Just into its own function? Or are you wanting it out of the file entirely?

Reply:
I think sync_job_runner.py is a good place for this method.